_posts/2016-08-26-babili.md

```diff
@@ -7,7 +7,7 @@ categories: announcements
 share_text: "Babili (babel-minify)"
 ---
 
-We released [Babili](https://github.com/babel/babili) as beta (0.0.1) a few days ago under an MIT license!
+We released [Babili](https://github.com/babel/minify) as beta (0.0.1) a few days ago under an MIT license!
```

Comment:
Think it's fine to leave the blog posts alone for historical purposes, the repo redirects anyway.